src/OrleansRuntime/Placement/IActivationSelector.cs

```diff
@@ -9,10 +10,13 @@ internal interface IActivationSelector
     {
         Task<PlacementResult> OnSelectActivation(
             PlacementStrategy strategy, GrainId target, IPlacementRuntime context);
+
+        bool TrySelectActivationSynchronously(
```

Comment:
Could this be made a single method with a return value of `ValueTask<PlacementResult>` instead of two methods?